### PR TITLE
Add port numbers for custom domain

### LINKED ISSUE #13: Serverless protocol and port config properties conflict with PS Template
`custom_data: {
    serverless_functions_protocol: 'http',
    serverless_functions_port: '3001',`

The PS Project Template uses the same variables so when you run in localhost the appConfig.js values override the PS Template features as well.

### PR DESCRIPTION
**Pull request by chaosloth**
### Summary

Add custom domain port numbers in AppConfig for this project

### Checklist

- [ ] Tested changes end to end
- [ ] Updated documentation
- [ ] Requested one or more reviewers

### CODE REVIEW

File: flex-template-future-cc-demo/public/appConfig.example.js [hunk 1]
```diff
@@ -8,8 +8,9 @@ var appConfig = {
   showSupervisorDesktopView: true,
   custom_data: {
     serverless_functions_protocol: 'http',
-    serverless_functions_port: '3001',
+    serverless_functions_port: '',
```
Comment (JasonLearmouth) [inline flex-template-future-cc-demo/public/appConfig.example.js:11]: I reckon this line should be deleted. It isn't used in this project any more.

File: flex-template-future-cc-demo/public/appConfig.example.js [hunk 2]
```diff
@@ -8,8 +8,9 @@ var appConfig = {
   showSupervisorDesktopView: true,
   custom_data: {
     serverless_functions_protocol: 'http',
```
Comment (JasonLearmouth) [inline flex-template-future-cc-demo/public/appConfig.example.js:10]: What's the reason that this property didn't get renamed? It is also overriding the PS Template features from --included-remote.

File: flex-template-future-cc-demo/public/appConfig.example.js (hunk 1 above)
Comment (chaosloth) [reply flex-template-future-cc-demo/public/appConfig.example.js:11]: agree

**Review by JasonLearmouth — APPROVED** [review]
Looks good now.

**Pull request merged**